src/components/Map/index.js

```diff
@@ -251,4 +254,11 @@ CaptureClick.propTypes = {
   setMarker: PropTypes.func.isRequired,
 };
 
+const GestureHandlingSetter = () => {
+  const map = useMap();
+  map.gestureHandling.enable();
+  map.addHandler('gestureHandling', GestureHandling);
```

Comment:
this is really something that you can't do with react components - every render means this function gets executed every time, and we are not (nor shouldn't be) in control of when renders happen - that's a React internal.

Effectively, running this kind of initialization during the render is a bad idea - it belongs in a `useEffect` that only runs once, or better (from https://github.com/elmarquis/Leaflet.GestureHandling/issues/65), it appears you can run:

```js
L.Map.addInitHook("addHandler", "gestureHandling", GestureHandling);
```
this looks a lot like a global, so perhaps we can even move that out of react alltogether (and put it in `src/map/index.js`) and handle it similarly to `fixIconUrls`.